### ISSUE TITLE
use ppi data in unsupervised mode, but can not  converge！ need help

### ISSUE THREAD

**Issue by homehehe** [williamleif/GraphSAGE#150]
the trend of loss:   who can tell me why????

Epoch: 0001
Iter: 0000 train_loss= 2.28088 train_mrr= 0.39205 train_mrr_ema= 0.39205 val_loss= 2.36228 time= 1.15651
Iter: 0050 train_loss= 2.37790 train_mrr= 0.07485 train_mrr_ema= 0.35128 val_loss= 2.36228 time= 0.02849
Iter: 0100 train_loss= 2.31257 train_mrr= 0.30818 train_mrr_ema= 0.31404 val_loss= 2.36228 time= 0.01703
Iter: 0150 train_loss= 2.34312 train_mrr= 0.11648 train_mrr_ema= 0.29781 val_loss= 2.36228 time= 0.01318
Iter: 0200 train_loss= 2.34605 train_mrr= 0.07353 train_mrr_ema= 0.27681 val_loss= 2.28311 time= 0.01132
Iter: 0250 train_loss= 2.26551 train_mrr= 0.12962 train_mrr_ema= 0.27216 val_loss= 2.28311 time= 0.01010
Iter: 0300 train_loss= 2.20238 train_mrr= 0.31675 train_mrr_ema= 0.26658 val_loss= 2.28311 time= 0.00932
Iter: 0350 train_loss= 2.14746 train_mrr= 0.22286 train_mrr_ema= 0.25806 val_loss= 2.28311 time= 0.00872
Iter: 0400 train_loss= 2.17668 train_mrr= 0.26786 train_mrr_ema= 0.25654 val_loss= 2.21059 time= 0.00834
Iter: 0450 train_loss= 2.17388 train_mrr= 0.17841 train_mrr_ema= 0.24381 val_loss= 2.21059 time= 0.00796
Iter: 0500 train_loss= 2.15469 train_mrr= 0.33229 train_mrr_ema= 0.24230 val_loss= 2.21059 time= 0.00768
Iter: 0550 train_loss= 2.13787 train_mrr= 0.20762 train_mrr_ema= 0.25058 val_loss= 2.21059 time= 0.00743
Iter: 0600 train_loss= 2.05377 train_mrr= 0.34222 train_mrr_ema= 0.24788 val_loss= 2.15021 time= 0.00726
Iter: 0650 train_loss= 2.16601 train_mrr= 0.15316 train_mrr_ema= 0.23802 val_loss= 2.15021 time= 0.00707
Iter: 0700 train_loss= 2.03879 train_mrr= 0.37843 train_mrr_ema= 0.25796 val_loss= 2.15021 time= 0.00693
Iter: 0750 train_loss= 2.10829 train_mrr= 0.27148 train_mrr_ema= 0.23796 val_loss= 2.15021 time= 0.00678
Iter: 0800 train_loss= 2.12745 train_mrr= 0.12650 train_mrr_ema= 0.24822 val_loss= 2.12446 time= 0.00667
Iter: 0850 train_loss= 2.06832 train_mrr= 0.47053 train_mrr_ema= 0.25520 val_loss= 2.12446 time= 0.00656
Iter: 0900 train_loss= 2.10655 train_mrr= 0.13009 train_mrr_ema= 0.24280 val_loss= 2.12446 time= 0.00649
Iter: 0950 train_loss= 2.05646 train_mrr= 0.39455 train_mrr_ema= 0.24797 val_loss= 2.12446 time= 0.00642

**Comment by richardsun** [comment]
I meet the same problem. have you solved it ?
